src/util/ScopeComponent.js

```diff
@@ -0,0 +1,29 @@
+/**
```

Comment:
Seems to be duplicate to https://github.com/nextcloud/vue-components/pull/70/files#diff-9f8488750cb07585c51b688fd9d06562 :wink: 

But the push approach is better than just overwriting it. :+1:

Reply:
Weird bug! My renaming seemed to have copied instead! :thinking: 
Nice catch!